_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -734,6 +734,15 @@ public static function reset_jetpack_options( $data ) {
 					) );
 					break;
 
+				case 'connection' :
+					if ( Jetpack::clear_all_connection_options() ) {
```

Comment:
The default of the switch(), which would be telling a lie `return new WP_Error( 'invalid_param', esc_html__( 'Invalid Parameter', 'jetpack' ), array( 'status' => 404 ) );`